Subject: Link libvdpao with libX11 since it uses symbols from it
Author: Russ Allbery <rra@debian.org>
Forwarded: no

libvdpau uses the symbols:

    _XEatData
    _XReply
    _XFlush
    _XReadPad
    XFree

which are provided by libX11, but wasn't linking with it directly, resulting
in warnings during the package build (and possibly errors later with better
linkers).

[Patch taken from
https://anonscm.debian.org/cgit/pkg-nvidia/libvdpau.git/tree/debian/patches/link-with-libx11.patch.]

Signed-off-by: Damien Lanson <damien@kal-host.com>

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -22,7 +22,8 @@ endif
 libvdpau_la_LIBADD = \
     $(DLOPEN_LIBS) \
     $(PTHREAD_LIBS) \
-    $(XEXT_LIBS)
+    $(XEXT_LIBS) \
+    $(X11_LIBS)
 
 libvdpau_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
